docs: Reorganize user guide (#2050)
webrecorder/browsertrix: docs/user-guide/getting-started.md

# Your first crawl

> We'll walk you through your first crawl. Open up a webpage that you'd like to crawl and note the URL for later.

Reviewer: ```suggestion
Let’s do your first crawl! Start by opening up a webpage that you'd like to crawl, and note the URL for later.
```

Author: `Let's crawl your first webpage`?

Reviewer: Even better!
